src/messagequeue/adapters/eventhub.ts

```diff
@@ -23,8 +23,42 @@ export const create = (
     await producer.close();
   };
 
+  const sendMessagesBatch = async(messages: Message[], batchOptions: CreateBatchOptions) => {
+    try {
```

Comment:
I'm not really a fan of huge try / catch statements that cover an entire function. I would reduce this to only the code  it really affects which is like the `batch.tryAdd` portion

Reply:
For this one, I wanted to cover the CreateBatch(), TryAdd() and SendBatch() functions. Because the file is so small it ends up being almost the entire function, but I cant try to put some operations outside the try